RoFILib/configuration/Generators.h

```diff
@@ -0,0 +1,93 @@
+#include <Configuration.h>
+#include <optional>
+#include <unordered_set>
+#include <vector>
+
+#ifndef ROBOTS_GENERATORS_H
+#define ROBOTS_GENERATORS_H
+
+    std::optional<Configuration> executeIfValid(const Configuration& config, const Action &action);
```

Comment:
Why the indent in the whole file?

Reply:
My bad, result of these functions being methods previously